binary_search_tree/binary_search_tree.py

```diff
@@ -1,17 +1,76 @@
+'''
```

Comment:
Btw, if you paste this comment inside a class or a function definition. it can be used as a `docstring` and help you document your programme. check out - https://www.python.org/dev/peps/pep-0257/